### PR TITLE
feat: create issue form for Press items

### LINKED ISSUE #187: Create Issue Form for new Press Release posting
Fields:

* Title (used in list views)
* Heading (if different from Title)
* Google doc link to content of Press Release, or external link to Press Release outside of Cal-ITP
* Tag: _currently `Contactless Payments`, `Benefits`, `GTFS`_
* Date: day, month, year
* Location: _default is `Sacramento, CA`_

### PR DESCRIPTION
**Pull request by machikoyasuda**
closes #187

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new_press.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+name: Request to add a new press release or press link
+description: Use this issue to request to add a new press release or press link to Calitp.org
+title: "New press request"
+labels: press
```
Comment (machikoyasuda) [inline .github/ISSUE_TEMPLATE/new_press.yml:4]:
```suggestion
labels:
  - content
  - press
```

File: .github/ISSUE_TEMPLATE/new_press.yml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      options:
+        - Contactless payments
+        - Benefits
+        - GTFS
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Date - Year
+      description: The date is used for press releases. Not required for external press links.
```
Comment (thekaveman) [inline .github/ISSUE_TEMPLATE/new_press.yml:41]: We need the date even for external press releases, it shows up on the listing page.

File: .github/ISSUE_TEMPLATE/new_press.yml [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+      multiple: false
+      options:
+        - 2023
+        - 2022
+        - 2021
+        - 2020
+  - type: dropdown
+    attributes:
+      label: Date - Month
+      description: The date is used for press releases. Not required for external press links.
```
Comment (thekaveman) [inline .github/ISSUE_TEMPLATE/new_press.yml:51]: We need the date even for external press releases, it shows up on the listing page.

File: .github/ISSUE_TEMPLATE/new_press.yml [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+        - "08"
+        - "09"
+        - "10"
+        - "11"
+        - "12"
+  - type: input
+    id: date
+    attributes:
+      label: Date
+      description: The date is used for press releases. Not required for external press links.
```
Comment (thekaveman) [inline .github/ISSUE_TEMPLATE/new_press.yml:70]: Since there is a separate Year and Month field, I think this is duplicative right?

**Review by thekaveman — CHANGES_REQUESTED** [review]

**Comment by machikoyasuda** [comment]
Tested out the new issue template here:

image
image

Form: https://github.com/machikoyasuda/kindara-client-gem/issues/new?assignees=&labels=content%2Cpress&projects=&template=new_press.yml&title=New+press+request

Created this issue: https://github.com/machikoyasuda/kindara-client-gem/issues/8 (The labels aren't on there b/c this repository doesn't have those labels).

File: .github/ISSUE_TEMPLATE/new_press.yml [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+        - "07"
+        - "08"
+        - "09"
+        - "10"
+        - "11"
+        - "12"
+  - type: input
+    id: location
+    validations:
+      required: true
```
Comment (thekaveman) [inline .github/ISSUE_TEMPLATE/new_press.yml:76]:
```suggestion
```

We can make `location` optional since we have a default value

**Review by thekaveman — APPROVED** [review]

**Pull request merged**